containers/zammad/docker-entrypoint.sh

```diff
@@ -5,6 +5,7 @@ set -e
 : "${AUTOWIZARD_JSON:=''}"
 : "${ELASTICSEARCH_HOST:=zammad-elasticsearch}"
 : "${ELASTICSEARCH_PORT:=9200}"
+: "${ELASTICSEARCH_PROTO:=http}"
```

Comment:
': "${ELASTICSEARCH_SSL_VERIFY:=true}"' is missing...

Reply:
I'll add it for documentation reasons but it's not strictly necessary?

Reply:
fixed